deltachat-rpc-client/examples/echobot.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+import asyncio
+import logging
+import sys
+
+import deltachat_rpc_client as dc
+
+
+async def main():
+    rpc = await dc.start_rpc_server()
+    deltachat = dc.Deltachat(rpc)
+    system_info = await deltachat.get_system_info()
+    logging.info("Running deltachat core %s", system_info["deltachat_core_version"])
+
+    accounts = await deltachat.get_all_accounts()
+    account = accounts[0] if accounts else await deltachat.add_account()
+
+    await account.set_config("bot", "1")
+    if not await account.is_configured():
+        logging.info("Account is not configured, configuring")
+        await account.set_config("addr", sys.argv[1])
+        await account.set_config("mail_pw", sys.argv[2])
+        await account.configure()
+        logging.info("Configured")
+    else:
+        logging.info("Account is already configured")
+        await deltachat.start_io()
+
+    async def process_messages():
+        fresh_messages = await account.get_fresh_messages()
+        fresh_message_snapshot_tasks = [
+            message.get_snapshot() for message in fresh_messages
+        ]
+        fresh_message_snapshots = await asyncio.gather(*fresh_message_snapshot_tasks)
```

Comment:
I am not deep in async but I remember reading somewhere that this could have issues with reaching limit of system file-descriptors or something like that if there is a lot of tasks, here there could be thousands of fresh messages so I guess issues could happen?

Reply:
maybe do the `get_snapshot()` in the loop below